### DISCUSSION TITLE
Map/minimap

### DISCUSSION THREAD

**Discussion by AdamTadeusz** [NeotokyoRebuild/neo#664] [Ideas]
Could add an ingame map toggleable with an m key, or a minimap that sits in one corner of the screen.

The hud element would map in game player's position to a pixel on the map. The map can draw multiple images one on top of another, starting with layers below the player with a low opacity, the current layer with a high opacity and finally upper layers with again a low opacity.

Entities already visible to the player in other hud elements like the ghost marker and friendly player markers could also be drawn on the map.

Here's a mockup of what it could look like

![mapIdea](https://github.com/user-attachments/assets/e1d9e3cf-44c2-4627-9e70-6a958ca1105e)

Big thanks to Wakfuck for the isometric image of oliostain

**Comment by brysondev** [comment]
I mean considering how many `_a` maps exist in the comp AND casual servers. I'd say a backlog item if at all.
